controllers/message-utils.js

```diff
@@ -1,20 +1,97 @@
-var db = require('../db');
-
-var updateCouchDB = function(record_id, body, callback) {
-  db.medic.updateWithHandler(
-    'medic',
-    'update_message_task',
-    record_id,
-    body,
-    callback
-  );
-};
+const _ = require('underscore'),
+      db = require('../db');
 
-var getTaskMessages = function(options, callback) {
+const getTaskMessages = function(options, callback) {
   db.medic.view('medic', 'tasks_messages', options, callback);
 };
 
+// copied from kujua-utils
+// TODO: get rid of this copy once Milan's refactor is complete
+const setTaskState = function(task, state, details) {
+  task.state = state;
+  task.state_details = details;
+  task.state_history = task.state_history || [];
+  task.state_history.push({
+    state: state,
+    state_details: details,
+    timestamp: new Date().toISOString()
+  });
+};
+
+const getTaskForMessage = function(uuid, doc) {
+  const getTaskFromMessage = (tasks, uuid) => tasks.find(task => {
```

Comment:
i think either this should not be an inner-function, or you could remove the `uuid` param.